jaeger-tracerresolver/README.md

```diff
@@ -0,0 +1,35 @@
+# Jaeger Tracer Resolver
+
+This module provides a Jaeger implementation for the [TracerResolver](https://github.com/opentracing-contrib/java-tracerresolver). This mechanism provides a vendor neutral approach for obtaining a `Tracer` using the JDK
+`ServiceLoader`.
+
+
+## Maven Dependency
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-tracerresolver</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
+## Usage
+
+```java
+Tracer tracer = TracerResolver.resolveTracer();
+```
+
+## Configuration Options
+
+The following configuration properties can be provided either as an environment variable or system property.
+The system property will override an environment variable for the same property name.
+
+Property | Default | Description
+--- | --- | ---
+JAEGER_SERVICE_NAME | _none_ | The service name (must be defined)
+JAEGER_AGENT_UDP_MAX_PACKET_SIZE | 65000 | The maximum packet size when communicating with the agent via UDP
```

Comment:
agreed - better to just say required/optional. The defaults will be applied by Configuration class.